features/steps/app_steps.rb

```diff
@@ -17,8 +17,8 @@
   # This step should only be used when the app has crashed, but the notifier needs a little
   # time to write the crash report before being forced to reopen.  From trials, 2s was not enough.
   sleep(5)
-  case Maze.driver.capabilities['platformName']
-  when 'Mac'
+  case Maze::Helper.get_current_platform
```

Comment:
I'm no Ruby expert, but shouldn't this just be `Maze::Helper.current_platform` ?

https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Naming/AccessorMethodName

Reply:
Maybe, I wasn't sure - it's not really an accessor in the getter/setter sense, more of a "work_out_the_current_platform" routine, if that works sense.  The whole `Helper` class hasn't reached a coherent state as yet - it's more a holding area for reusable routines and a step-up from things living outside of any module, but it could certainly be evolved more in the future.  Once it's gather a certain amount of momentum it would make sense to refactor it into a more targeted set of classes.